CHANGELOG.md

```diff
@@ -26,6 +26,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### [shopsys/framework]
 #### Changed
+- [#272 - Changed concept of Components](https://github.com/shopsys/shopsys/pull/143):
```

Comment:
let us discuss the changelog entry personally :slightly_smiling_face: What about upgrading notes?